executor/api/payload/model_metadata.go

```diff
@@ -0,0 +1,9 @@
+package payload
+
+type ModelMetadata struct {
```

Comment:
Why new file instead of inside metadata.go?

Reply:
Different package - this way I was able to avoid circular dependency.

Reply:
Ah perfect, thanks for clarifying, if this is required may be worth adding a comment to mention why it's being added in a separate file?

Reply:
Yes, I will add it. Basically it now has a meaning of deserialized payload and is used as output on corresponding method of the `SeldonApiClient` interface

Reply:
Added a proper note.